docs/bclinux-installation-guide.md

```diff
@@ -0,0 +1,8 @@
+# Installing Intel® Clear Containers 3.0 on BC-Linux* version 7
+## Introduction of BC-Linux
+[BC-Linux](https://www.bclinux.org/) is an enterprise Linux operating system developed by China Mobile for cloud computing and big data users.
+BC-Linux has developed some common server operating systems based on the CentOS OS Open Source Community Edition. For example, BC-Linux 7.3 is based on the same source code as Red Hat Enterprise RHEL 7.3 and maintains 100% compatibility with RHEL.
+Additionally, BCLinux has released some customized versions of the server OS for different scenarios. These customized versions simplify the common BC-Linux 7 system and integrate the packages required for the cloud computing and big data scenarios. BCLinux-for-DCOS is a good representative. It uses a customized high-version kernel, fixes the security vulnerabilities in the low-version kernel, and adds multiple optimization patches for the container.
+
+## Installation steps
+Clear Containers 3.0 is available for BC-Linux 7.3, 7.2, and BCLinux-for-DCOS-V1.0. BC-Linux is basically a variant of CentOS so users can refer to docs/centos-installiation-guide.md to install Clear Containers 3.0 on BC-Linux* version 7.
```

Comment:
Ah - I've just noticed that "docs/centos-installiation-guide.md" needs to be a link.